exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -230,6 +275,8 @@ class OTLPExporterMixin(
 
     def __init__(
         self,
+        stub: ExportStubT,
+        result: ExportResultT,
```

Comment:
My understanding of the current code is that it expects `self._stub()` and `self._result()` to be defined in the child class. If you wanted to avoid breaking change in the signature, I think we could add `@abstractmethod`s for stub and result as well?

Although IMO the code is easier to understand with the change in this PR.